.github/workflows/reusable_build-push.yml

```diff
@@ -54,16 +54,18 @@ jobs:
                       }).join(',');
 
             - name: Build and push
-              uses: docker/build-push-action@v3
+              uses: docker/build-push-action@v6
               with:
-                  context: .
-                  file: ./Dockerfile
-                  pull: true
-                  push: true
-                  platforms: "${{ inputs.platforms }}"
-                  build-args: |
-                      BUILD_DATE=${{ steps.date.outputs.date }}
-                      VCS_REF=${{ github.sha }}
-                      BRANCH='${{ github.ref }}'
-                      TAG='${{ github.ref }}'
-                  tags: "${{ steps.tags.outputs.result }}"
+                context: .
+                file: ./Dockerfile
+                pull: true
+                push: true
+                platforms: "${{ inputs.platforms }}"
+                build-args: |
+                  BUILD_DATE=${{ steps.date.outputs.date }}
+                  VCS_REF=${{ github.sha }}
+                  BRANCH='${{ github.ref }}'
+                  TAG='${{ github.ref }}'
+                tags: "${{ steps.tags.outputs.result }}"
+                sbom: true
```

Comment:
Does this work for java / go / python? Where is the sbom visible?

Reply:
What about repos that don't build a dockerfile?

Reply:
I believe so. You can inspect it in the attestations page and with the docker cli.
For repos that don't have a dockerfile, docker cannot generate an sbom. So we could add a dockerfile to that repo..  Do you have an example core service that doesn't have a dockerfile?

Reply:
I'm thinking more of support repos like java_common and java_kidl. Also a possibility for kb_sdk_plus if GraalVM works out

Reply:
We could use this then

https://github.com/aquasecurity/trivy-action?tab=readme-ov-file#using-trivy-to-generate-sbom

Reply:
> github attestations page I believe 

Ok, so we'd have to run a workflow before I could look at the result

Reply:
This also seems like something to look into https://docs.github.com/en/actions/security-for-github-actions/using-artifact-attestations/using-artifact-attestations-to-establish-provenance-for-builds#generating-build-provenance-for-binaries